### PR TITLE
[CDE-826] - DashboardComponent: Expose a way to turn on and off the

### PR DESCRIPTION
**Pull request by eliofreitas**
parameter propagation

### CODE REVIEW

**Comment by eliofreitas** [comment]
@pamval please review

**Comment by pamval** [comment]
@jvelasques please review

**Comment by eliofreitas** [comment]
Building pull request

**Comment by brosander** [comment]
http://wingman.pentaho.com:8080/job/wingman/23/

**Comment by eliofreitas** [comment]
Building pull request

**Comment by eliofreitas** [comment]
Building pull request

**Comment by brosander** [comment]
http://wingman.pentaho.com:8080/job/wingman/24/

**Comment by eliofreitas** [comment]
Building pull request

**Comment by eliofreitas** [comment]
Building pull request

**Comment by eliofreitas** [comment]
Building pull request

**Comment by brosander** [comment]
http://wingman.pentaho.com:8080/job/wingman/27/

**Comment by eliofreitas** [comment]
Building pull request

**Comment by brosander** [comment]
http://wingman.pentaho.com:8080/job/wingman/28/

File: cde-pentaho5/resource/resources/custom/amd-components/Dashboard/DashboardComponent.js [hunk 1]
```diff
@@ -113,6 +124,26 @@ define([
       });
     },
 
+    /**
+     * @summary Pauses parameter propagation.
+     * @description <p>Method that causes parameters to stop propagating</p>.
+     *              <p>To resume propagation {@link cdf.components.DashboardComponent#resumePropagation|resumePropagation} should be called.</p>
+     * @see {@link cdf.components.DashboardComponent#_pause|_pause}
```
Comment (jvelasques) [inline cde-pentaho5/resource/resources/custom/amd-components/Dashboard/DashboardComponent.js]: we probably don't want to link to private properties

**Comment by jvelasques** [comment]
@pamval I just have that comment on the private properties, other than that looks good

**Comment by eliofreitas** [comment]
updated @jvelasques @pamval

**Pull request merged**